### PR TITLE
fixed reciprocal

### LINKED ISSUE #2: Fix reciprocal function
The reciprocal function doesn't work !1!111

### PR DESCRIPTION
**Pull request by yunijeong5**
Please check my code

### CODE REVIEW

File: calculator.js [hunk 1]
```diff
@@ -1,78 +1,77 @@
...
+	display.value = display.value ** 2;
 }
 
 // BUG: FIX reciprocal function
 function calculateReciprocal() {
-    // Hint: Reciprocal is 1/x
-    display.value = 1-value;
+	// Hint: Reciprocal is 1/x
+	let display = document.getElementById("display");
+	display.value = 1 / value;
```
Comment (awyuan) [inline calculator.js]: Value doesn't work because it isn't defined, you probably wanted it to be display.value instead.

**Review by awyuan — CHANGES_REQUESTED** [review]
See listed comments on the calculator js file

File: calculator.js (hunk 1 above)
Comment (yunijeong5) [reply calculator.js]: thank you!

**Review by awyuan — APPROVED** [review]
Looks great!

**Pull request merged**